scripts/quests/crystalWar/A_Little_Knowledge.lua

```diff
@@ -182,7 +182,26 @@ quest.sections =
 
         [xi.zone.THE_ELDIEME_NECROPOLIS_S] =
         {
-            ['Erlene'] = quest:event(15):replaceDefault(),
```

Comment:
We still need this replaceDefault event if the below block is not applied

Reply:
Done

Reply:
I seem to remember this needing more than just completed quest.  Isn't there something along the lines of either having to be SCH, or having SCH at least to level 5?

Reply:
the canLearnSpell does not do the check?

Reply:
Yep, you're absolutely correct